### ISSUE TITLE
WCAG本文:「WCAG 2.2 関連文書」の見直し

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag22#1533]
ID: `wcag-2-2-supporting-documents`
[対象箇所](https://waic.jp/translations/WCAG22/#wcag-2-2-supporting-documents)
[原文](https://www.w3.org/TR/WCAG22/#wcag-2-2-supporting-documents)

**Comment by momdo** [comment]
機械翻訳の候補から：
補足文書、サポート文書、補助文書

あるいは<https://eow.alc.co.jp/search?q=Supporting+Document>から類推して：
付属文書

補助文書とするのはどうか？ひととおり見直してみたけど、こうだとどうか…？

---

> WCAG 2.2 の文書は、安定した参照可能な技術標準を必要とする人たちのニーズを満たすように作成されている。関連文書と呼ばれるその他の文書は、WCAG 2.2 文書に基づいて、WCAG が新しい技術にどのように適用されるかを説明するために更新できるようにすることを含め、その他の重要な役割を果たすものである。関連文書には以下に挙げるものがある:

> The WCAG 2.2 document is designed to meet the needs of those who need a stable, referenceable technical standard. Other documents, called supporting documents, are based on the WCAG 2.2 document and address other important purposes, including the ability to be updated to describe how WCAG would be applied with new technologies. Supporting documents include:

WCAG 2.2 文書は、安定した参照可能な技術標準を必要とする人々のニーズを満たすように設計されている。補助文書と呼ばれるその他の文書は、WCAG 2.2 文書を基礎としており、WCAG を新しい技術にどのように適用するかを説明するための更新を含む、その他の重要な目的に対応する。補足文書には次が含まれる:

> How to Meet WCAG 2.2 - WCAG 2.2 のカスタマイズ可能なクイックリファレンス。コンテンツ制作者がウェブコンテンツを制作したり評価したりする際に用いるガイドライン、達成基準、テクニックのすべてが含まれる。これには WCAG 2.0、2.1 及び 2.2 のコンテンツが含まれており、関連するコンテンツにコンテンツ制作者がフォーカスできるよう、様々な方法でフィルタリングできるようになっている。

> How to Meet WCAG 2.2 - A customizable quick reference to WCAG 2.2 that includes all of the guidelines, success criteria, and techniques for authors to use as they are developing and evaluating web content. This includes content from WCAG 2.0, 2.1, and 2.2, and can be filtered in many ways to help authors focus on relevant content.

How to Meet WCAG 2.2 - WCAG 2.2 のカスタマイズ可能なクイックリファレンスで、コンテンツ制作者がウェブコンテンツの開発及び評価に使用できる、ガイドライン、達成基準、及びテクニックの全てを含む。これには、WCAG 2.0、2.1 及び 2.2 のコンテンツが含まれており、コンテンツ制作者が関連するコンテンツに集中できるよう、様々な方法でフィルタリング可能である。

> Understanding WCAG 2.2 - WCAG 2.2 を理解して実践するための解説書。重要なトピックスとあわせて、WCAG 2.2 の各ガイドライン及び達成基準を「理解する」ための簡潔な文書がある。

> Understanding WCAG 2.2 - A guide to understanding and implementing WCAG 2.2. There is a short "Understanding" document for each guideline and success criterion in WCAG 2.2 as well as key topics.

Understanding WCAG 2.2 - WCAG 2.2 を理解して実践するためのガイド。WCAG 2.2 の各ガイドライン及び達成基準、並びに重要なトピックについて、短い「理解」文書がある。

> Techniques for WCAG 2.2 - テクニック集及びよくある失敗例集。個々に別々の文書になっており、解説、事例、コード例、テストが含まれる。

> Techniques for WCAG 2.2 - A collection of techniques and common failures, each in a separate document that includes a description, examples, code and tests.

Techniques for WCAG 2.2 - テクニック及びよくある失敗例のコレクション。個別の文書には、説明、事例、コード、検証が含まれる。

> The WCAG 2 Documents - WCAG 2 の関連文書及び補足的なガイダンスの簡単な紹介。

> The WCAG 2 Documents - A brief introduction to the WCAG 2 supporting documents and supplemental guidance.

The WCAG 2 Documents - WCAG 2 の補助文書及び補足ガイダンスの簡潔な紹介。

> What's New in WCAG 2.2 では、アクセシビリティ問題の例を説明するペルソナともに、新しい達成基準を紹介している。

> What's New in WCAG 2.2 introduces the new success criteria with persona quotes that illustrate the accessibility issues.

What's New in WCAG 2.2 では、アクセシビリティの問題を説明するペルソナの引用文とともに、新しい達成基準を紹介している。

> WCAG 2.2 に関する教育資料を含む関連資料の説明は、Web Content Accessibility Guidelines (WCAG) Overview を参照。例えば、ウェブアクセシビリティのビジネスにおける効果、ウェブサイトのアクセシビリティを改善するための実施計画作成、及びアクセシビリティ方針といったトピックに関する補足資料は、WAI Resources に挙げられている。

> See Web Content Accessibility Guidelines (WCAG) Overview for a description of the WCAG 2.2 supporting material, including education resources related to WCAG 2. Additional resources covering topics such as the business case for web accessibility, planning implementation to improve the accessibility of websites, and accessibility policies are listed in WAI Resources.

WCAG 2 に関連する教育資料を含む、WCAG 2.2 の補足情報の説明については、Web Content Accessibility Guidelines (WCAG) Overview を参照。ウェブアクセシビリティのビジネスケース、ウェブサイトのアクセシビリティ向上のための実施計画、アクセシビリティポリシーなどのトピックを扱う追加資料は、WAI Resources に掲載されている。

**Comment by caztcha** [comment]
@momdo ありがとうございます。やはり、そうですよね...。(自分も、ability の訳出、難しいなと思って、上記の案もあまり自信はなかったです。)

https://github.com/waic/wcag22/issues/1533#issuecomment-3976715582 のもんどさん案で、PR 作成しようと思います。
